### ISSUE TITLE
Blockwise ensemble meta-estimator

### ISSUE THREAD

**Issue by TomAugspurger** [dask/dask-ml#135]
Fit clones of an estimator on each block / partition, and ensemble the results somehow http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.VotingClassifier.html, http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.BaggingClassifier.html#sklearn.ensemble.BaggingClassifier, http://scikit-learn.org/stable/modules/generated/sklearn.ensemble.BaggingRegressor.html#sklearn.ensemble.BaggingRegressor

**Comment by js3711** [comment]
2nd this for tree based methods as well.  
http://scikit-learn.org/stable/modules/ensemble.html#ensemble
Namely ExtraTrees, and RandomForest based approaches

This might involve changing how the current dask_ml cross validation methods work as well.  It would be nice to add the individual "voters" to the task graph.

**Comment by jakirkham** [comment]
+1 for RandomForest :)

Also would add AdaBoost and Gradient Tree Boosting.

**Comment by TomAugspurger** [comment]
Closed by #657.
